addons/gunbag/functions/fnc_toGunbagCallback.sqf

```diff
@@ -40,6 +43,6 @@ _unit removeWeapon _weapon;
 _gunbag setVariable [QGVAR(gunbagWeapon), [_weapon, _items, _magazines], true];
 
 // play sound
-if (!isNil "ACE_Backpacks") then {
+if (["ace_backpacks"] call EFUNC(common,isModLoaded)) then {
```

Comment:
It checks actual CfgPatches rather than just global variable. We also use that everywhere. If it's not needed, why do we have that function in the first place?